get-started/the-stack.md

```diff
@@ -7,97 +7,147 @@ mapped_pages:
   - https://www.elastic.co/guide/en/elastic-stack/current/overview.html
 products:
   - id: elastic-stack
-  - id: kibana
 applies_to:
+  serverless:
   stack:
 ---
+# {{stack}} and the {{search-platform}}
 
-# The {{stack}}
+The [{{search-platform}}](https://www.elastic.co/platform) is the full suite of products and features available in {{ecloud}} including its unique orchestration, autoscaling, and cloud-native features.
```

Comment:
I think that we still need to talk about this, but I don't agree with defining this strictly in relation to elastic cloud. the page you link to also doesn't do this - it covers on prem installs too

<img width="1246" height="632" alt="image" src="https://github.com/user-attachments/assets/3ebd235a-a4cb-4d53-97a9-461e03724175" />

Reply:
@georgewallace want to draw your attention to [Liam's comment](https://github.com/elastic/docs-content/pull/2922#issuecomment-3586621065) (specifically item 2) where he is more aligned with this current framing

Reply:
Yeah, I like the focus on the AI capabilities.. If we do talk about these expanded capabilities here at the forefront it would be imperative that we point folks to the right place to get started with that stuff. Core pages like this are great but no one is going to read the entire page. So if we add things like vector db, sematic search, eis, geo, whatever else we want to talk about then we just need to have a great place to land them. Similar to my squirrel comment on another thread. I am not sure I agree with not talking about the platform for self managed.. I think no matter the deployment type the platform is key to everything. I'll add this as a comment in the thread as well.